### PR TITLE
[NUI] Add GetWindowId()

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Add GetWindowId()

### API Changes ###
nothing

### CODE REVIEW

**Comment by dongsug-song** [comment]
This issue is pending. 
If needed, this will be re-opened later.

**Comment by dongsug-song** [comment]
this will be merged after native patches are merged.

File: src/Tizen.NUI/src/public/Window.cs [hunk 1]
```diff
@@ -1117,6 +1117,18 @@ public void SetAvailableOrientations( List<Window.WindowOrientation> orientation
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Get native window ID
+        /// </summary>
+        /// <returns>native window ID</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int GetWindowId()
```
Comment (hyunjushin) [inline src/Tizen.NUI/src/public/Window.cs:1125]: We already provides GetNativeHandle. I think GetNativeId is better.

Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Window.cs]: Hello, I fixed it as following comments. please review. thank you.

**Pull request merged**